Makefile

```diff
@@ -189,7 +189,7 @@ bump-kubevirtci:
 	./hack/bump-kubevirtci.sh
 
 fossa:
-	hack/dockerized "FOSSA_TOKEN_FILE=${FOSSA_TOKEN_FILE} ./hack/fossa.sh"
+	hack/dockerized "FOSSA_TOKEN_FILE=${FOSSA_TOKEN_FILE} PULL_BASE_REF=${PULL_BASE_REF} CI=${CI} ./hack/fossa.sh"
```

Comment:
Does this change the behavior of fossa lanes?

Reply:
It just adds the base ref, which is normally auto-determined by FOSSA, but since Prow rebases on HEAD each time it runs a job, it can't determine the base ref. So we are adding it here. This is also the reason why we add the env vars that are set by prow here.

See [docs for env vars on prow jobs](https://docs.prow.k8s.io/docs/jobs/#job-environment-variables)